### PR TITLE
Add accessor functions for comment and stop signal

### PR DESCRIPTION
**Pull request by baude**
Podman commit needs the ability to set a comment and stop signal
when committing a container to an image.  Adding set/get functions
for each.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by baude** [comment]
sit on this one while i work some edge case issues

**Comment by baude** [comment]
Fixed up, should be good now.

File: config.go [hunk 1]
```diff
@@ -579,3 +579,29 @@ func (b *Builder) SetDomainname(name string) {
 func (b *Builder) SetDefaultMountsFilePath(path string) {
 	b.DefaultMountsFilePath = path
 }
+
+// Comment returns the comment which will be set in the container and in
+//containers built using images buiilt from the container
```
Comment (TomSweeneyRedHat) [inline config.go]: nit: buiilt to built ditto line 597

Comment (baude) [reply config.go]: it is a copy of existing text but can do

**Comment by rhatdan** [comment]
@baude Waiting for the spelling error fix and then we can merge.

File: config.go (hunk 1 above)
Comment (baude) [reply config.go]: fixed

**Comment by baude** [comment]
fixed spelling error, please merge

File: config.go [hunk 2]
```diff
@@ -579,3 +579,29 @@ func (b *Builder) SetDomainname(name string) {
...
+//containers built using images built from the container
+func (b *Builder) Comment() string {
+	return b.Docker.Comment
+}
+
+// SetComment sets the Comment which will be set in the container and in
+// containers built using images built from the container.
+func (b *Builder) SetComment(comment string) {
+	b.Docker.Comment = comment
+	b.OCIv1.History[0].Comment = comment
```
Comment (nalind) [inline config.go]: The `History` list can be empty if the image is being built from scratch, and when it isn't empty, this is the entry for the base layer.

Comment (baude) [reply config.go]: Adding check for len ... I think adding the comment to the base layer is OK; at least that is what I have seen being done before.  I'll suggest a work-around in my next push.  or is there an alternate way you would want the comment set for v1?

File: config.go [hunk 3]
```diff
@@ -579,3 +579,30 @@ func (b *Builder) SetDomainname(name string) {
 func (b *Builder) SetDefaultMountsFilePath(path string) {
 	b.DefaultMountsFilePath = path
 }
+
+// Comment returns the comment which will be set in the container and in
+//containers built using images built from the container
```
Comment (nalind) [inline config.go]: Add a whitespace after `//` here, to match the rest.

**Comment by nalind** [comment]
Whitespace nit, otherwise LGTM.

File: config.go (hunk 3 above)
Comment (baude) [reply config.go]: fixed

**Comment by nalind** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**